packages/common/mui/src/lib/data-table-cells/arrow-indicator-cell/arrow-indicator-cell.tsx

```diff
@@ -33,18 +34,15 @@ export const ArrowIndicatorCell = ({
     },
   } = cell;
   const { palette, shape, spacing } = useTheme();
-  const colorOnUp = greenOnArrowUp ? palette.primary.light : palette.error.main;
-  const colorOnDown = greenOnArrowUp
+  const arrowColor = tooRisky
     ? palette.error.main
+    : greenOnArrowUp
```

Comment:
You still need to maintain the logic in the if block below. So:

greenOnArrowUp && arrowUp => green
greenOnArrowUp && !arrowUp => orange
!greenOnArrowUp && arrowUp => orange
!greenOnArrowUp && !arrowUp => green

Reply:
Or basically, you can add back the `colorOnUp` and `colorOnDown` variables back in.